doc/acting/discontinued/teb/motion_planner.py

```
@@ -0,0 +1,1020 @@
+            distance_obj (float): distance to overtake object
+        """
+        pose_list = self.trajectory.poses
+
+        # Only use fallback
+        self.generate_overtake_trajectory(distance_obj, pose_list)
+        self.__overtake_status = 1
+        self.overtake_success_pub.publish(self.__overtake_status)
+        return
+
+    def __direction_vector_to_quaternion(
+        self, direction, reference=np.array([0, 0, 1])
```

Ruff (0.8.2)

Do not perform function call `np.array` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)